quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/DockerKeycloakDistribution.java

```diff
@@ -158,6 +194,19 @@ public void setRequestPort(int port) {
         }
     }
 
+    public void copyProvider(String groupId, String artifactId) {
```

Comment:
and possibly even here, although is probably less likely to happen:
```
if (providerPath == null) {
        throw new RuntimeException("Failed to resolve provider artifact: " + groupId + ":" + artifactId);
    }
```

Reply:
Looking at `resolveArtifact` method it is not possible the result will be `null` (famous last words :D :D) Anyway, I would skip the suggestion. I would say the main point here is to detect if something goes wrong so we don't falsely think the provider was correctly copied while it was not. Given checks in `resolveArtifact` and calling `Files.isRegularFile` we should end up with an exception if the file was not found so I believe this is good enough. WDYT?